google-cloud-language/lib/google/cloud/language.rb

```diff
@@ -28,6 +28,10 @@ module Cloud
     # recognition, and text annotations.
     # - [Product Documentation][]
     #
+    # *This release, 0.28.0, introduces breaking changes relative to the previous
```

Comment:
Do you want to future-proof the phrasing with something like:

> The 0.28.0 release introduced breaking changes relative to the previous

Reply:
Yeah, that's a good idea. Done.